schemas/2016-09-01/Microsoft.Resources.json

```diff
@@ -43,6 +43,16 @@
           "enum": [
             "Microsoft.Resources/deployments"
           ]
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "The subscription to deploy to"
+        },
+        "resourceGroup": {
```

Comment:
the "resourceGroup" property is supported in api-version >= 2017-05-10

Reply:
Made changes to https://github.com/Azure/azure-resource-manager-schemas/blob/66ccf729436c29b6e2bc8860036fa1f19525ccba/generator/processors/Microsoft.Resources.ts - please take a look